### PR TITLE
Fix ESLint errors + JSDoc comments for scripts/

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR fixes up the issues reported by ESLint, as well as updates the JSDoc comments, and performs just a little bit of refactoring for efficiency and consistency in the scripts files.

### CODE REVIEW

File: scripts/fix-browser-order.js [hunk 1]
```diff
@@ -1,28 +1,29 @@
-#!/usr/bin/env node
```
Comment (ExE-Boss) [inline scripts/fix-browser-order.js]: This&nbsp;is&nbsp;correct for&nbsp;this&nbsp;file.

```suggestion
#!/usr/bin/env node
```

File: scripts/fix-feature-order.js [hunk 2]
```diff
@@ -1,7 +1,20 @@
-#!/usr/bin/env node
```
Comment (ExE-Boss) [inline scripts/fix-feature-order.js]: ditto

```suggestion
#!/usr/bin/env node
```

**Comment by ddbeck** [comment]
This PR appears to partly relate to #5252 and https://github.com/mdn/browser-compat-data/pull/5635#issuecomment-603854873.

Like #5655, I'd suggest closing this and breaking it into separate PRs. There's a few things that this PR changes which don't seem strictly related: JSDoc comments, whether scripts should be executable, and copyright notices. It'd be a lot easier to review them separately. Thank you!

**Comment by vinyldarkscratch** [comment]
Sounds good to me, I'll start breaking this down then!  I'll mark this as "not ready" and leave it open until the separate PRs have been created.

**Pull request closed without merging**